src/alias_dirs.rs

```diff
@@ -80,7 +80,7 @@ pub fn print_keys_long_colored(ad: &AliasDirs, key_style: &Style, min_number_of_
 
     for (key, val) in ad {
         // don't use `println!` to avoid overhead of flushing each time
```

Comment:
Please take a look at the comment. `println!` invokes flushing of stdout that makes printing slower, so I use `print!` instead.